### ISSUE TITLE
Player panel locations may fail WCAG AA compliance due to bad contrast ratios.

### ISSUE THREAD

**Issue by MiniOranges** [MiniOranges/starwatch#40]
An example would be:
![image](https://user-images.githubusercontent.com/56451580/121560645-57a02300-c9dd-11eb-99f1-2d7aa507964e.png)

Fails WCAG just slightly:
![image](https://user-images.githubusercontent.com/56451580/121560702-6555a880-c9dd-11eb-99e8-7d8a50f54573.png)

Adjusting this to a higher contrast ratio will help users with visual impairments use the application.
